tests/tests_pytorch/loggers/test_all.py

```diff
@@ -157,64 +156,6 @@ def log_metrics(self, metrics, step):
         assert log_metric_names == expected
 
 
-@pytest.mark.parametrize("logger_class", ALL_LOGGER_CLASSES_WO_NEPTUNE)
-def test_loggers_save_dir_and_weights_save_path_all(tmpdir, monkeypatch, logger_class):
-    """Test the combinations of save_dir, weights_save_path and default_root_dir."""
-
-    with contextlib.ExitStack() as stack:
-        for mgr in LOGGER_CTX_MANAGERS:
-            stack.enter_context(mgr)
-        _patch_comet_atexit(monkeypatch)
-        _test_loggers_save_dir_and_weights_save_path(tmpdir, CometLogger)
-
-
-def _test_loggers_save_dir_and_weights_save_path(tmpdir, logger_class):
```

Comment:
We could keep the test and replace all `weights_save_path` with `ModelCheckpoint(dirpath=...)`.
But then all this test does is assert that model checkpoint dirpath is correct and that it is different from loggers save_dir. Since we already have tests for each of these, there is basically no value in keeping it.